src/index.js

```diff
@@ -28,4 +28,9 @@ const RootedApp = root(tree, App);
 window.tree=tree;
 window.events=events;
 
-ReactDOM.render(<RootedApp />, document.querySelector('#app'));
+superagent.get('/api/user').then((res) => {
+  const user = JSON.parse(res.text);
+  window.STATE_FROM_SERVER = {user};
+
+  ReactDOM.render(<RootedApp />, document.querySelector('#app'));
+})
```

Comment:
see https://github.com/drone/drone/blob/master/server/template/files/index.html for more details. We render the `index.html` server-side and inject the `user` and a `csrf-token` before serving to the user, avoiding additional round-trips / page flickering.

we just need a way to ensure we only load the user from the API in development mode, and not in production mode.